Bottom: fdcd409937c3eefb05df4ebef6b69bf539202279
Top:    82da22afe8643a13004c2debf86cdf5b00e14358
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-20 13:50:55 +0800

Refresh of tun

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 1f1624f..04331de 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -2615,19 +2615,6 @@ struct socket *tun_get_socket(struct file *file)
 }
 EXPORT_SYMBOL_GPL(tun_get_socket);
 
-struct skb_array *tap_get_skb_array(struct file *file)
-{
-	struct tun_file *tfile;
-
-	if (file->f_op != &tun_fops)
-		return ERR_PTR(-EINVAL);
-	tfile = file->private_data;
-	if (!tfile)
-		return ERR_PTR(-EBADFD);
-	return &tfile->tx_array;
-}
-EXPORT_SYMBOL_GPL(tap_get_skb_array);
-
 module_init(tun_init);
 module_exit(tun_cleanup);
 MODULE_DESCRIPTION(DRV_DESCRIPTION);
